srcpkgs/amule/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'amule'
+pkgname=amule
+version=11066
+revision=1
+wrksrc=amule-master
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man
+	--enable-cas --enable-wxcas --enable-amule-daemon
+	--enable-amulecmd --enable-amule-gui --enable-alc
+	--enable-alcc --enable-webserver --disable-debug
+	--enable-optimize --enable-geoip --disable-upnp
+	--enable-fileview --with-boost  --with-wx-config=/usr/bin/wx-config-gtk3"
+hostmakedepends="tar autoconf automake gettext gettext-devel wxWidgets-gtk3"
+makedepends="boost-devel zlib-devel wxWidgets-gtk3-devel gd-devel geoip-devel crypto++-devel"
+depends="wxWidgets-gtk3-devel gd-devel geoip-devel crypto++-devel"
+short_desc="Emule-like client for the eD2k and Kademlia networks"
+maintainer="hazayan <hazayan@hotmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/amule-project/amule"
+distfiles="https://github.com/amule-project/amule/archive/master.tar.gz"
```

Comment:
master is wrong, if you really need to use unreleased tarballs, use a git revision/hash.
If you are using the master tarball, the checksum is going to break as soon as they add new
commits to master and we won't be able to rebuild the package without updating it.

Reply:
The upstream project doesn't define proper release tags. I replaced master with the latest commit on the repo.